### ISSUE TITLE
DatePicker: Arrow icons for changing time are not rendered - e.g. css class: ui-icon-carat-1-n

### ISSUE THREAD

**Issue by jxmai** [primefaces/primefaces#7631]
**Describe the defect**

Arrow icons in datepicker component are not rendered properly. 

**Reproducer**

Go to showcase: https://www.primefaces.org/showcase/ui/input/datepicker/datePicker.xhtml

![image](https://user-images.githubusercontent.com/10467831/126885301-decfc6f1-9c70-4457-9fa9-a35c365ecbac.png)

![image](https://user-images.githubusercontent.com/10467831/126885312-b9f35de2-a696-447e-a538-c10bfc384475.png)

If we take a look at the console, we can see the css classes are there but nothing is rendered

![image](https://user-images.githubusercontent.com/10467831/126885367-00e796d1-037a-48b1-bebe-02a47cf83460.png)

**Environment:**
 - PF Version: _10.0_
 - JSF + version: _e.g. Mojarra 2.2.20 or MyFaces 2.3.9_
 - Affected browsers: _e.g. Chrome, iOS, ALL_

**Expected behavior**
The icon should be rendered properly

for example, https://api.jqueryui.com/1.12/theming/icons/

![image](https://user-images.githubusercontent.com/10467831/126885336-237d4c88-a503-4139-aa9c-fa8758dddfa4.png)

**Comment by melloware** [comment]
I think this might be a Saga Vela Arya theme issue.

**Comment by christophs78** [comment]
https://github.com/primefaces/primefaces/issues/7114 ?

**Comment by jxmai** [comment]
Interesting. The text indent in jquery ui is strange but this line https://github.com/primefaces/primefaces/blob/53e480465f9009f5223f75cbdd66af9f2e7b8dde/src/main/resources/META-INF/resources/primefaces/jquery/jquery.ui.css#L353 has already been there since 11 Aug 2017

That looks like the same defect. I included the css https://github.com/primefaces/primefaces/commit/366b62afd7c55ac91c239f69ad6124001426909b into the showcase page and can see the page icons are rendered.

![image](https://user-images.githubusercontent.com/10467831/126902264-ba85efaa-4414-4dcf-aba6-75f74747cecb.png)

---

Some thoughts: In the latest theme, icon.scss is removed and I doubt that would the culprit to expose the defect. But as far as this ticket is concerned, it is fixed. However, the direct cause could be the latest theme changes, but we adjusted in PF codebase instead.
